components/math/MultiplicationBlockPlayer2.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+
+export interface MultiplicationBlockPlayer2Props {
```

Comment:
Don't think we want this duplicate component. It's pretty much the same as your other one except it shows blue.

Could you use the player prop with your existing ternary to achieve the same thing?